src/transform/md.ts

```diff
@@ -12,6 +12,7 @@ import extractTitle from './title';
 import getHeadings from './headings';
 import sanitizeHtml, {defaultOptions, sanitizeStyles} from './sanitize';
 import {olAttrConversion} from './plugins/ol-attr-conversion';
+import inlineCode from './plugins/inline-code';
```

Comment:
It's already added in DefaultPlugins. If it doesn't work correctly we need to figure out why rather than add the plugin again.